lib/Serialization/Deserialization.cpp

```diff
@@ -43,6 +43,7 @@
 #include "clang/AST/Attr.h"
 #include "clang/Basic/SourceManager.h"
 #include "clang/Basic/AttributeCommonInfo.h"
+#include "clang/Index/USRGeneration.h"
```

Comment:
This might mean that we now need to link `swiftSerialization` against `clangIndex` – should be a simple CMake script change if true

Reply:
Good point!